src/test/java/com/training/sample/tests/SampleTest.java

```diff
@@ -1,6 +1,10 @@
 package com.training.sample.tests;
```

Comment:
Please do not touch the sample tests and sample pages.
All your web based tests should be created in the package - web/tests/
and all web based pages should go in package - web/pages/
Also use proper indentation for the code.